@types/premid/index.d.ts

```diff
@@ -102,17 +102,17 @@ interface BasePresenceData {
 	buttons?: [ButtonData, ButtonData?];
 }
 
-interface ListeningPresenceData extends BasePresenceData {
-	type: ActivityType.Listening;
+interface MediaPresenceData extends BasePresenceData {
+	type: ActivityType.Listening | ActivityType.Watching;
 	largeImageText?: string | Node;
 }
 
-interface NonListeningPresenceData extends BasePresenceData {
+interface NonMediaPresenceData extends BasePresenceData {
 	type?: Exclude<ActivityType, ActivityType.Listening>;
```

Comment:
Hello,
Shouldn't the ActivityType.Watching be excluded from the NonMedia interface? Like this?

```suggestion
	type?: Exclude<ActivityType, ActivityType.Listening | ActivityType.Watching>;
```